server/src/main/java/com/talkka/server/common/dto/ApiResponseDto.java

```diff
@@ -0,0 +1,18 @@
+package com.talkka.server.common.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Getter
+@Builder
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+public class ApiResponseDto<T> {
```

Comment:
```suggestion
public class ApiResponseDto<T> {
```
저희 클래스명 ApiRespDto로 정한걸로 기억하는데 확인 한번 부탁드립니다!

Reply:
감사합니다 변경하겠습니다.